collector/lib/NetworkStatusNotifier.cpp

```diff
@@ -326,15 +335,27 @@ void NetworkStatusNotifier::AddConnections(::google::protobuf::RepeatedPtrField<
       // the case where we send a close event for a connection that it doesn't
       // know about.
       //
-      if (!rate_limiter.Allow(delta_entry.first.container())) {
-        CLOG_THROTTLED(INFO, std::chrono::seconds(5)) << "Rate limiting connections reported from container " << delta_entry.first.container();
+      std::string container = delta_entry.first.container();
+      if (!rate_limiter.Allow(container)) {
+        // [] operator will default construct the map element
+        // if it does not exist
+        rate_limited_containers[container]++;
         COUNTER_INC(CollectorStats::net_conn_rate_limited);
         continue;
       }
     }
 
+    added_events++;
     updates->AddAllocated(conn_proto);
   }
+
+  if (rate_limited_containers.size() != 0) {
+    for (auto container : rate_limited_containers) {
+      CLOG(INFO) << "Rate limited " << container.second << " connections from container " << container.first;
+    }
+  }
+
+  CLOG(DEBUG) << "Processed " << total_events << " events; sending " << added_events;
```

Comment:
Since the total events are the same as  all the elements in the `delta` map and a `ConnMap` is just a renamed `std::unordered_map`, you don't really need `total_events`, you can just call `delta.size()`.
```suggestion
  CLOG(DEBUG) << "Processed " << delta.size() << " events; sending " << added_events;
```

Reply:
Done